nixos/modules/services/misc/bees.nix

```diff
@@ -0,0 +1,93 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.beesd;
+
+  fsOptions = with types; {
+    options.spec = mkOption {
+      type = str;
+      description = ''
+        Specification usable by findmnt; ie. LABEL=foo, UUID=bar, etc.
+      '';
+    };
+    options.hashTableSizeMB = mkOption {
+      type = types.addCheck types.int (n: mod n 16 == 0);
+      default = 1024; # 1GB; default from upstream beesd script
+      description = ''
+        Hash table size in MB; must be a multiple of 16.
+
+        A larger ratio of index size to storage size means smaller blocks of duplicate content are recognized.
+
+        If you have 1TB of data, a 4GB hash table (which is to say, a value of 4096) will permit 4KB extents (the smallest possible size) to be recognized, whereas a value of 1024 -- creating a 1GB hash table -- will recognize only aligned duplicate blocks of 16KB.
```

Comment:
The formatting here should be fixed. Lines should be under 80 columns long, newlines don't result in linebreaks. If you want a linebreak:

```nix
''
  blablabla (there is no line break following this sentence)
  foobarbaz
  </para>
  <para>
  ^^ that just was a linebreak
''
```